build.proj

```diff
@@ -255,8 +255,8 @@
     <CallTarget Targets="BuildSetup" Condition=" '$(SkipHelp)' == 'false' "/>
     <CallTarget Targets="CodeSignInstaller"
                 Condition=" '$(CodeSign)' == 'true' and '$(Scope)' == 'all'" />
-    <Message Importance="high" Text="Running Static Analyser" />
-    <CallTarget targets="DependencyAnalysis" ContinueOnError="ErrorAndContinue" />
+    <!--<Message Importance="high" Text="Running Static Analyser" />
```

Comment:
I don't think you are supposed to modify this file.

Reply:
Yes; I'm doing so to bypass the local build problem. Will revert this finally.

Reply:
ok